spark/core.py

```diff
@@ -455,23 +455,24 @@ def parse_color(self, func_name, *args):
 
         if argc == 1:
             if type(args[0]) is int:
-                return "rgb({}, {}, {})".format(args[0], args[0], args[0])
+                return "rgb({}, {}, {})".format(*np.clip([args[0]] * 3))
```

Comment:
Won't this try to clip the argument between itself and itself? or fail due to lack of positional argument?

Reply:
yep, fixed. never mind!